plugins/chat/assets/stylesheets/common/chat-message-collapser.scss

```diff
@@ -19,8 +19,9 @@ $max_video_height: 150px;
   }
 
   .chat-video-upload {
-    height: $max_video_height;
-    width: calc(#{$max_video_height} / 9 * 16);
+    max-height: $max_video_height;
+    width: calc(#{$max_video_height} / 9 * 16) !important;
```

Comment:
oh no, `!important` 🥲 can we avoid any of those three uses? shifting stuff around, increasing specificity of selectors, etc?

Reply:
Looks like I never staged the last minute changes. It seems more `!irrelevant` to me :upside_down_face:
I also remove the other ones